plot_data/core.py

```diff
@@ -490,29 +497,31 @@ def mpl_plot(self, ax=None, edge_style=None):
         else:
             edge_style = DEFAULT_EDGESTYLE
 
-        ax.plot([self.point1[0], self.point2[0]], [self.point1[1], self.point2[1]], **edge_style.mpl_arguments())
+        ax.plot([self.point1[0], self.point2[0]], [self.point1[1], self.point2[1]], **edge_style.mpl_arguments(),
+                **kwargs)
         return ax
 
 
 class LineSegment(LineSegment2D):
+    """ A line segment. This is a primitive that can be called by PrimitiveGroup. """
+
     def __init__(self, data: List[float], edge_style: EdgeStyle = None, name: str = ''):
         # When to remove support?
```

Comment:
I think we can remove the support of this. We could the time when we deprecated this, but I think it is quite old

Reply:
Removec